source/server/spongineer/logs.rst

```diff
@@ -9,6 +9,128 @@ logfiles from SpongeForge and SpongeVanilla servers including short descriptions
    :depth: 2
    :local:
 
+Configure Logging
+=================
+
+Sometimes plugins log too much information and thus spam the logs with useless messages and thus hide more important
```

Comment:
bias much? I think no reason to mention that here, what is useful is always in the eye of the beholder
most of the time logging was added for a reason, if it's really pointless one should instead open a ticket with the plugin and ask them to remove the specific logging